dependencies.yaml

```
@@ -321,11 +343,13 @@ dependencies:
         matrices:
           - matrix: {cuda: "12.*"}
             packages: &build_python_packages_cu12
-              - &rmm_cu12 rmm-cu12==24.8.*
+              - &rmm_cu12 rmm-cu12==24.8.*,>=0.0.0a0
```

This anchor appears unused.
```suggestion
              - rmm-cu12==24.8.*,>=0.0.0a0
```

---

Thanks for catching that, will remove it.